Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -1360,6 +1366,9 @@ def start_auth(self, complete_command: str) -> str:
 and enter the code **{user_code}** to authenticate.
 2. Run the **{complete_command}** command in the War Room."""
 
+    def is_auth_code_reconfigured(self) -> bool:
+        return not get_integration_context().get('auth_code') == self.auth_code
```

Comment:
What happens if the `auth_code` parameter is not used?
Apparently `get_integration_context().get('auth_code')` will be equal to `None` and self.auth_code will be `''`,
Can you please check it?

Reply:
you are absolutely right!
I made some changes have a look :)